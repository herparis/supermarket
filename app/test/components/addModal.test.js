import React from 'react'
import { shallow } from 'enzyme'

import AddModal from '../../src/components/addModal/addModal'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('AddModal components rendeding', () => {
  const addModal = shallow(<AddModal />)
  it('addModal renders', () => {
    checkRender(addModal)
  })
})