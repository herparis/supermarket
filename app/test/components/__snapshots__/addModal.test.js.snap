// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddModal components rendeding addModal renders 1`] = `ShallowWrapper {}`;
